.travis/build_package.sh

```diff
@@ -117,13 +117,13 @@ cd $ROOT
       EXTRA_CXX_FLAGS="-Werror=inconsistent-missing-override"
       ;;
   esac
-  
-  
+
+
   mytime cmake -DCMAKE_CXX_FLAGS="${CXX_FLAGS} ${EXTRA_CXX_FLAGS}" -DCGAL_DONT_OVERRIDE_CMAKE_FLAGS:BOOL=ON -DBUILD_TESTING=ON -DWITH_tests=ON -DWITH_examples=ON -DWITH_demos=$WITHDEMOS ..
-  mytime ctest -j2 -L $ARG --output-on-failure
-  
+  mytime ctest -j2 -L $ARG -E execution___of__ --output-on-failure
```

Comment:
If we are sure `$ARG` does not have trailing spaces, then we can modify the regex thay way:
```suggestion
  mytime ctest -j2 -L $ARG'([_][A-Z]|$)' -E execution___of__ --output-on-failure
```

See the difference I have on my machine:
```
[lrineau@bonnard]~/Git/cgal-master/build% ctest -L 'Triangulation' --print-labels 
Test project /home/lrineau/Git/cgal-master/build
All Labels:
  AABB_tree_Examples
  Triangulation_2_Demo
  Triangulation_2_Examples
  Triangulation_2_Tests
  Triangulation_3_Demo
  Triangulation_3_Examples
  Triangulation_3_Geomview_demos_Demo
  Triangulation_3_Tests
  Triangulation_Examples
  Triangulation_Tests
[lrineau@bonnard]~/Git/cgal-master/build% ctest -L 'Triangulation([_][A-Z]|$)' --print-labels
Test project /home/lrineau/Git/cgal-master/build
All Labels:
  AABB_tree_Examples
  Triangulation_Examples
  Triangulation_Tests
```

The presence of `AABB_tree_Examples` is because of the the test `check_build_system`.